docs/content/latest/reference/configuration/yb-master.md

```diff
@@ -341,6 +335,24 @@ The number of replicas, or copies of data, to store for each tablet in the unive
 
 Default: `3`
 
+##### --yb_num_shards_per_tserver
+
+The number of shards per YB-TServer for each YCQL table when a user table is created.
+
+Default: `-1` (server internally sets default value). For servers with two or less CPU cores, then the default value is `4`. For four or more CPU cores, the default value is `8`. If set using `yb-ctl` or `yb-docker-ctl`, then the default value is `2`.
```

Comment:
This value can be overriden on a per table basis using `WITH tablets = <num>` clause of `CREATE TABLE` statement.

Reply:
Added note.